.github/workflows/image-publish.yml

```diff
@@ -7,14 +7,13 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-name: image-publish-insiders
+name: image-publish
 
 on:
   push:
     branches: 
       - main
-    tags:
-      - '^7\.[0-9]+\.[0-9]+$'
+  workflow_dispatch:
```

Comment:
I think you need to set some variables here or this block will fail. In particular, if we're triggering this by hand, don't we have to set a branch/tag value to release?

Reply:
see this action:

* https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/.github/workflows/plugin-registry-build-publish-content-gh-pages.yaml#L21-L26 support user input override for default
* https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/.github/workflows/plugin-registry-build-publish-content-gh-pages.yaml#L37-L55 set an env var depending on if we have user input or the default 
* https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/.github/workflows/plugin-registry-build-publish-content-gh-pages.yaml#L65-L68 use the value defined by default or user input